src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -342,8 +342,8 @@ GcsActorManager::GcsActorManager(
       actor_gc_delay_(RayConfig::instance().gcs_actor_table_min_duration_ms()) {
   RAY_CHECK(worker_client_factory_);
   RAY_CHECK(destroy_owned_placement_group_if_needed_);
-  actor_state_counter_.reset(
```

Comment:
Creating a shared pointer with `new` leads to two allocation, while only once for `std::make_shared`.
Ref: modern effective C++ item 21